app/models/actor.rb

```diff
@@ -0,0 +1,3 @@
+class Actor < ApplicationRecord
+    belongs_to :production, polymorphic: true
```

Comment:
When creating a new `actor` from the controller, the Rails will throw an exception because in Actor model there is a `belongs_to production` association. The form will expect to have a production when submitted.
To mitigate this, just add the option: `optional: true`.

Read more here: https://stackoverflow.com/questions/38584189/error-validation-failed-images-imageable-must-exist-rails-5-0-paperclip-5
```suggestion
    belongs_to :production, polymorphic: true, optional: true
```

Reply:
Btw, this is probably what the tutorial suggested you to do so I understand if you weren't aware of this issue.